tools/cmd/docs/transformations.go

```diff
@@ -53,7 +53,7 @@ func setDisableAutoGenTag(cmd *cobra.Command) {
 }
 
 func markExperimenalToAPICommands(cmd *cobra.Command) {
-	cmd.Short = "`experimental <https://www.mongodb.com/docs/atlas/cli/current/command/atlas-api/>`_: " + cmd.Short
+	cmd.Short = "`Public Preview: please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli, see more at <https://www.mongodb.com/docs/atlas/cli/current/command/atlas-api/>`_: " + cmd.Short
```

Comment:
I think you now created a link that links to the docs, but you want it to link to the feedback form right?